src/components/command-bar/components/activator/command-bar-activator.module.css

```diff
@@ -41,6 +41,13 @@
 			display: none;
 		}
 	}
+
+	@nest html[data-theme='dark'] & {
```

Comment:
Note, it is our goal to minimize these types of overrides. Since the nav is a special case, given that it needs to stay 'dark' in light mode, there are more overrides than should be expected with other components.